tests/nosetests/pyanaconda_tests/simple_ui_test.py

```diff
@@ -56,6 +56,10 @@ def hubs(self):
     def action_classes(self):
         return self.interface._collectActionClasses(self.paths["spokes"], StandaloneSpoke)
 
+    @property
+    def ordered_action_classes(self):
+        return self.interface._orderActionClasses(self.action_classes, self.hubs)
```

Comment:
Why don't we just fix the method `_orderActionClasses`?

Reply:
See my answer below in https://github.com/rhinstaller/anaconda/pull/2435#issuecomment-614055745.